.github/workflows/build_and_publish.yaml

```diff
@@ -0,0 +1,48 @@
+name: Build and upload to PyPI
+on: [push]
+
+jobs:
+  build_sdist:
+    name: Build source distribution
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: ${{ matrix.PYTHON_VERSION }}
```

Comment:
Why does this variable expansion work? I don't see any `matrix` attribute defined for the job.